terminologies/phenotype/wild_type.jsonld

```diff
@@ -1,5 +1,5 @@
 {
-  "@id": "https://openminds.ebrains.eu/controlledTerms/phenotype/wild_type",
+  "@id": "https://openminds.ebrains.eu/controlledTerms/phenotype/wildType",
   "@type": "https://openminds.ebrains.eu/controlledTerms/Phenotype",
   "name": "wild type",
```

Comment:
I've checked again. It's completely valid to use the notation "wildtype"
@olinux again, not your mistake but could you fix it ? and change the at_id as well?